kitsune/questions/static/questions/js/components/AAQApp.jsx

```diff
@@ -39,17 +40,30 @@ export default class AAQApp extends React.Component {
       validationErrors: QuestionEditStore.getValidationErrors(),
       questionState: QuestionEditStore.getState(),
       userAuth: UserAuthStore.getAll(),
-      troubleshooting: TroubleshootingDataStore.getAll(),
+      troubleshooting: TroubleshootingDataStore.getAll()
```

Comment:
Trailing commas are actually fine in ES6 code. ES5 defined trailing commas in objects and lists as OK.